roles/openshift_logging_mux/tasks/main.yaml

```diff
@@ -177,6 +177,48 @@
   check_mode: no
   changed_when: no
 
+# storageclasses are used by default but if static then disable
+# storageclasses with the storageClassName set to "" in pvc.j2
```

Comment:
This task looks almost identical to the one which follows it.  The difference appears to be 'storage_class_name'.  Is it possible to reduce to a single task and selectively include the storage_class_name if its provided?